sites/partners/package.json

```diff
@@ -37,6 +37,7 @@
     "ag-grid-community": "^26.0.0",
     "axios": "^1.8.3",
     "axios-cookiejar-support": "4.0.6",
+    "class-validator": "^0.14.1",
```

Comment:
Now that we are only validating on the backend, we should be able to remove this import

Reply:
@ludtkemorgan 

1. i have merged `main` branch once again since there was new updates and ran `yarn generate:client` once again. Hope it looks ok now.

2. Removed.